steward/src/commands/keys/cosmos/delete.rs

```diff
@@ -3,22 +3,26 @@ use abscissa_core::{clap::Parser, Application, Command, Runnable};
 use signatory::FsKeyStore;
 use std::path::Path;
 
+/// Steward keys cosmos delete [name]
 #[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Delete a Cosmos Key.\n This command deletes a Cosmos key from your keystore when provided with the keyname."
+)]
 pub struct DeleteCosmosKeyCmd {
-    pub args: Vec<String>,
+    /// Cosmos keyname in keystore, takes a String
+    pub name: String,
 }
 
-/// The `gork keys cosmos delete [name] ` subcommand: delete the given key
+/// The `steward keys cosmos delete [name] ` subcommand: delete the given key
```

Comment:
Similarly this bit is probably not necessary for documentation.

Reply:
done!